### PR TITLE
Write doxygen for navfn

### PR DESCRIPTION
**Pull request by mlsdpk**
---

## Basic Info

| Info | Please fill out this column |
| ------ | ----------- |
| Ticket(s) this addresses   | (#2155 ) |
| Primary OS tested on | - |
| Robotic platform tested on | - |

---

## Description of contribution

* I added doxygen for navfn

### CODE REVIEW

File: nav2_navfn_planner/include/nav2_navfn_planner/navfn_planner.hpp [hunk 1]
```diff
@@ -37,52 +37,100 @@ namespace nav2_navfn_planner
...
+   * @return nav_msgs::Path of the generated path
+   */
   nav_msgs::msg::Path createPlan(
     const geometry_msgs::msg::PoseStamped & start,
     const geometry_msgs::msg::PoseStamped & goal) override;
 
 protected:
-  // Compute a plan given start and goal poses, provided in global world frame.
+  /**
+   * @brief // Compute a plan given start and goal poses, provided in global world frame.
```
Comment (SteveMacenski) [inline nav2_navfn_planner/include/nav2_navfn_planner/navfn_planner.hpp:90]:
```suggestion
   * @brief Compute a plan given start and goal poses, provided in global world frame.
```

**Review by SteveMacenski — CHANGES_REQUESTED** [review]
Just a quick change to remove some comment forward slashes. Otherwise looks really good to me, thank you for taking some time to work on this!

**Review by SteveMacenski — CHANGES_REQUESTED** [review]
Just a quick change to remove some comment forward slashes. Otherwise looks really good to me, thank you for taking some time to work on this!

**Comment by mlsdpk** [comment]
> Just a quick change to remove some comment forward slashes. Otherwise looks really good to me, thank you for taking some time to work on this!

Fixed! Thank you.

**Review by SteveMacenski — APPROVED** [review]

**Pull request merged**